File: frontend-mealsonwheels/src/types/user/MemberDTO.ts
// src/types/user/MemberDTO.ts

import type { UserDTO } from "./UserDTO";

export interface MemberDTO extends UserDTO {
  dietaryRestriction: string;
  address: string;
  memberLocationLat: number;
  memberLocationLong: number;
  caregiverId: number;
}

File: backend-mealsonwheels/src/main/java/com/merrymeal/mealsonwheels/dto copy done/roleDTOs/MemberDTO.java
package com.merrymeal.mealsonwheels_backend.dto.roleDTOs;

import com.merrymeal.mealsonwheels_backend.dto.UserDTO;
import lombok.*;
import lombok.experimental.SuperBuilder;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@SuperBuilder
public class MemberDTO extends UserDTO {
    private String dietaryRestriction;
    private String address;
    private double memberLocationLat;
    private double memberLocationLong;
    private Long caregiverId;
}
